src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -408,6 +415,7 @@ function ReportActionCompose({
                                     />
                                     <ComposerWithSuggestions
                                         ref={composerRef}
+                                        // @ts-expect-error TODO: Remove this once ComposerWithSuggestions is migrated to TypeScript.
```

Comment:
Please add a github issue link

Reply:
There is no issue for that in the spreadsheet